theforeman.org/pipelines/release/pipelines/foreman-rpm.groovy

```diff
@@ -42,7 +42,7 @@ pipeline {
         stage('Push RPMs') {
             agent { label 'admin && sshkey' }
             steps {
-                push_rpms_direct("foreman-nightly/RHEL/8", "nightly/el8")
+                push_rpms_direct("foreman-nightly/el8", "nightly/el8")
```

Comment:
This can now easily be rewritten to use `foreman_el_releases` to fully decouple this groovy file from the configuration, making the future transition to el9 easier.
```suggestion
                for (release in foreman_el_releases) {
                  push_rpms_direct("foreman-nightly/${release}", "nightly/${release}")
                }
```